src/Twig/SchemaOrgMetadataExtension.php

```diff
@@ -102,8 +112,20 @@ private function getType(Model $object)
         switch (true) {
             case $object instanceof ArticleVersion:
                 return 'ScholarlyArticle';
+            case $object instanceof BlogArticle:
+                return 'Blog';
```

Comment:
We could move `LabsPost` up to here as it should return `Blog` also:

```
case $object instanceof BlogArticle:
case $object instanceof LabsPost:
    return 'Blog';
```

Reply:
I thought these were supposed to be alphabetical?